public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -710,3 +788,7 @@ const getStyles = (theme: GrafanaTheme2) => ({
 });
 
 export { Policy };
+
+export function isAbleToSeeAutogeneratedChunk() {
+  return contextSrv.hasRole('Admin');
```

Comment:
What are the drawbacks of showing this autogenerated branch to `Editors` and `Viewers`? 
I can imagine that it can be a valuable debug tool for non-admins

Reply:
💯